doc/source/v2.0.0_migration.rst

```diff
@@ -55,55 +55,55 @@ getPlayer
+    The reason for this, is that these fields are all, in one way or another, included in the new work field 'currentWork'.
     Some of these values are also irrelevant.
-    Take a look at the new singularity.getCurrentWork function.
+    Take a look at the new singularity.getCurrentWork function:
 
     All fields ending in _mult have been moved to the 'mults' struct.
     For example: getPlayer().hacking_skill_mult => getPlayer().mults.hacking_skill
 
-    skills have been moved to the skills struct
+    skills has been moved to the skills struct
     For example: getPlayer().hacking => getPlayer().skills.hacking
 
-    exp have been moved to the exp struct
+    exp has been moved to the exp struct
     For example: getPlayer().hacking_exp => getPlayer().exp.hacking
 
-    hp have been moved to the hp struct
+    hp has been moved to the hp struct
     For example: getPlayer().max_hp => getPlayer().hp.max or hp.current
 
-    hasWseAccount, hasTixApiAccess, has4SData, has4SDataTixApi have been removed and replaced with similar stock functions
+    hasWseAccount, hasTixApiAccess, has4SData, has4SDataTixApi have been removed and replaced with similar stock functions.
 
 workForCompany
 --------------
 
-    The argument 'companyName' is now not-optional.
+    The argument 'companyName' is now required.
 
 
 getScriptIncome & getScriptExpGain
 ----------------------------------
 
-    Those 2 functions used to have a call where no arguments would return the total for all scripts. This caused weird signature.
-    If you want to get the total income/exp for all scripts used the new getTotalScriptIncome / getTotalScriptExpGain instead.
+    These two functions used to have a call where, if no arguments were provided, it would return the total for all scripts. This caused weird signature.
+    If you want to get the total income/exp for all scripts, use the new getTotalScriptIncome / getTotalScriptExpGain instead.
 
 scp
 ---
 
-    scp has it's 2 last argument reversed, the signature is now scp(files, destination, optional_source)
+    The last two arguments of spc have been reversed. The signature is now scp(files, destination, optional_source)
 
 Singularity
 -----------
 
-    A while ago top level singularity function were deprecated in favor of the singularity namespace.
+    The top level singularity functions were deprecated a while ago in favor of the singularity namespace.
     This means calls like 'ns.connect' need to be changed to 'ns.singularity.connect'
 
 
 stock.buy, stock.sell, stock.short
 ----------------------------------
     
-    These functions were renamed to stock.buyStock, stock.sellStock, and stock.buyShort because 'buy', 'sell', and 'short'
+    These functions were renamed to stock.buyStock, stock.sellStock, and stock.buyShort because 'buy', 'sell', and 'short',
     are very common tokens that would trick the ram calculation.
```

Comment:
This added comma is not correct

Reply:
Hah, I suppose the oxford comma is more a matter of taste. 

![Screenshot_20230429_161753_Chrome.png](https://user-images.githubusercontent.com/107779145/235326339-ca58e986-3380-429b-94e7-a274846f0423.png)

Some consider it to be not grammatically correct and redundant, others consider it to make more sense and reflect the parts of speech more clearly.

I tend to usually go for the oxford comma, but it won't bother me to leave it out either.

Reply:
Ooh it's English pedantics time! :)

Snarling is correct here; this is a different issue than Oxford comma.

Oxford comma is about whether or not to have a comma between the second-to-last item in a list, as your source notes. In the given sentence, it would be the difference between:
"... because 'buy', 'sell', and 'short' are very common tokens..."
and
"... because 'buy', 'sell' and 'short' are very common tokens..."
(2nd one is without the Oxford comma)

Snarling objected to this, which is *not* the Oxford comma:
""... because 'buy', 'sell', and 'short', are very common tokens..."

You could argue for a comma after the first list (before the "because"), but here it feels awkward.

Reply:
Oof, you guys are right! Thanks for explaining how I was wrong. I got it twisted here, and I didn't review the sentence after my change very carefully. I had probably assumed what I wrote was right.

Reply:
Sorry about that, I'll fix that up and remove the comma.